### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by Sibz** [zfzackfrost/iffy-rs#1]
Only thing I can think of is my lib version is using rust 2021 and nightly builds
```
error: expected `?`                                                                                  
   --> src\lib.rs:236:31
    |
236 |             iffy::i!(1 == 1 ? 2 : 3);

```
![image](https://user-images.githubusercontent.com/12050358/148646037-e24474d4-0047-4654-baa2-d8e2ed0fdfd7.png)

**Comment by lmburns** [comment]
Yeah, I'm unsure what the deal with this is. I cloned the repo and messed around with it and tried figuring out what the problem was, but I couldn't. I found that using commas or no delimiter at all compiled. I've also looked at other crates that use similar function-like procedural macros, which use uncustomary delimiters like `?` or `%` and they seem to get it to work. 

I'd really like for this crate to compile because I would use it all of the time, I just don't know very much about procedural macros. I would also add a feature of recursive ternary operators like this if I could:
```rust
let var = iffy::i!(condition1 ? someValue
            : condition2 ? anotherValue
            : defaultValue);
```

**Comment by zfzackfrost** [comment]
Hey sorry for the delay. I will take a look at it this weekend and let you know the status.
